### ISSUE TITLE
DeconvNet fine tuning

### ISSUE THREAD

**Issue by ckchng** [HyeonwooNoh/DeconvNet#8]
Thanks for making your work available, deeply appreciated. 

I'm hoping to fine tune your pre-trained DeconvNet, but I noticed that stage 003 make BN layers testable. Hence I assume that I should not load 'DeconvNet_trainval_inference.caffemodel' straightaway and starts the training, but I'm not sure. Please let me know if my concern is valid, if it does, can you please upload the caffemodel after stage 002 training? Thanks a lot.

**Comment by hussamullah** [comment]
Were you able to finetune deconvnet network and the FCN network for you own dataset? If yes, how did you do that.

**Comment by ckchng** [comment]
Yes I was able to do that. Regarding my question earlier on, I fine tune it using 'stage_2_train_result.caffemodel'. What have you tried and what's your problem?

**Comment by hussamullah** [comment]
Thanks for the reply. Can you please share your skype id on my email address: hussamullahkhan@gmail.com?  I am having the following problems:-
1-This implementation needs a trained FCN too, how were you able to train that? Because there is no code for its training in this branch
2- The format of training data for second stage is very odd and the pixels aren't labeled, instead it is binary image with just the boundries of the object. There is also a bounding box argument added to the train.txt file which has negative coordinates which quite confusing.
3- This implementation is using edge box proposals, wouldn't that make this implementation slow?

I'll be really glad if you can reply back asap. Thanks in advance.

**Comment by OrangePlusPlus** [comment]
@hussamullah Have you already solved your problem No.2 and No.3? I am confused by the same problems as you. If yes, how did you do that.
I'll be very glad if you can reply back. Thanks in advance.
